manifests/v1beta1/components/mysql/mysql.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     spec:
       containers:
         - name: katib-mysql
-          image: mysql:8
+          image: mysql:8.0.26
```

Comment:
Can we also add `imagePullPolicy: Always` here to avoid problems with caching data if users are running another `mysql` instance for other components ?
I remember that we had couple of problems with that.

Reply:
`imagePullPolicy: Always` is not required as long `mysql:8.0.26` tag is not updated, and cache can be used if available. 

However, I don't have any strong opinion, I can add it if you feel so.

Reply:
Do we have any use-case when `mysql` team can update previous image tag on the dockerhub ?

Reply:
Looking at [release history](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/My-sql])  & [dockerhub](https://hub.docker.com/_/mysql?tab=tags&page=1&ordering=last_updated) Mysql Team hasn't done it. I think it should be fine without `imagePullPolicy:"Always"`

A possible usecase would be when Dockerimages such as [ubuntu](https://hub.docker.com/_/ubuntu?tab=tags&page=1&ordering=last_updated&name=18.04) are rebuilt and pushed again to dockerhub when there are security fixes. Mysql doesn't seem to do so either.

Even if there were to be a security fix with mysql docker image. The mysql version would remain the same and this issue would not occur.
```
[InnoDB] Cannot boot server version 80017 on data directory built by version 80018. Downgrade is not supported
```

Reply:
Thank you for the explanation @munagekar!